modelmapper/src/main/java/com/bealdung/domain/Game.java

```diff
@@ -0,0 +1,74 @@
+package com.bealdung.domain;
```

Comment:
Typo in the package name, so you'll have to refactor/rename the directory and package (baeldung, not bealdung) for all the Java files.